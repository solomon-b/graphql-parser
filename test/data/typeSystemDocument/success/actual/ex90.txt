[ Right
    ( Left
        ( Right
            ( DirectiveDefinition
                { dirDefDescription = Nothing
                , dirDefName = "@example"
                , dirDefArguments = []
                , dirDefLocations =
                    [ Right TSDLFIELD_DEFINITION
                    , Right TSDLARGUMENT_DEFINITION
                    ]
                }
            )
        )
    )
, Right
    ( Left
        ( Left
            ( Right
                ( Left
                    ( Left
                        ( Left
                            ( Left
                                ( Right
                                    ( ObjectTypeDefinition
                                        { objectDescription = Nothing
                                        , objectName = "SomeType"
                                        , objectInterfaces = []
                                        , objectDirectives = []
                                        , objectFields =
                                            [ FieldDefinition
                                                { fieldDefDescription = Nothing
                                                , fieldDefName = "field"
                                                , fieldDefArgumentsDef =
                                                    [ InputValueDefinition
                                                        { inputValueDescription = Nothing
                                                        , inputValue = "arg"
                                                        , inputType = NamedType "Int"
                                                        , inputDefaultValue = Nothing
                                                        , inputValueDirectives =
                                                            [ Directive
                                                                { name = "@example"
                                                                , arguments = fromList []
                                                                }
                                                            ]
                                                        }
                                                    ]
                                                , fieldDefType = NamedType "String"
                                                , fieldDefDirectives =
                                                    [ Directive
                                                        { name = "@example"
                                                        , arguments = fromList []
                                                        }
                                                    ]
                                                }
                                            ]
                                        }
                                    )
                                )
                            )
                        )
                    )
                )
            )
        )
    )
]