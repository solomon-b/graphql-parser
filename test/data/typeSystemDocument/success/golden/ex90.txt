[ Right
    ( Left
        ( Right
            ( DirectiveDefinition
                { dirDefDescription = Nothing
                , dirDefName = "@example"
                , dirDefArguments = []
                , dirDefLocations =
                    [ Right TSDLFIELD_DEFINITION
                    , Right TSDLARGUMENT_DEFINITION
                    ]
                }
            )
        )
    )
, Right
    ( Left
        ( Left
            ( Right
                ( Left
                    ( Left
                        ( Left
                            ( Left
                                ( Right
                                    ( ObjectTypeDefinition
                                        { objectDescription = Nothing
                                        , objectName = "SomeType"
                                        , objectInterfaces = []
                                        , objectDirectives = []
                                        , objectFields =
                                            [ FieldDefinition
                                                { fieldDefDescription = Nothing
                                                , fieldDefName = "field"
                                                , fieldDefArgumentsDef =
                                                    [ InputValueDefinition
                                                        { inputValueDescription = Nothing
                                                        , inputValueName = "arg"
                                                        , inputValueType = NamedType "Int"
                                                        , inputValueDefault = Nothing
                                                        , inputValueDirectives =
                                                            [ Directive
                                                                { dName = "@example"
                                                                , dArguments = Nothing
                                                                }
                                                            ]
                                                        }
                                                    ]
                                                , fieldDefType = NamedType "String"
                                                , fieldDefDirectives =
                                                    [ Directive
                                                        { dName = "@example"
                                                        , dArguments = Nothing
                                                        }
                                                    ]
                                                }
                                            ]
                                        }
                                    )
                                )
                            )
                        )
                    )
                )
            )
        )
    )
]